core/src/rpc/clementine.proto

```diff
@@ -83,8 +83,24 @@ message WinternitzPubkey {
   repeated bytes digit_pubkey = 3;
 }
 
-// A deposit request's details.
 message DepositParams {
+  oneof deposit_data {
+    OriginalDeposit original_deposit = 1;
```

Comment:
I don't think OriginalDeposit is an appropriate name here, Original can be confused to mean "unique" when used on its own. Something like BaseDeposit may be better.